src/index.js

```diff
@@ -1,3 +1,4 @@
+/* global Cypress, cy */
```

Comment:
`lint-staged` forced me to add this

Reply:
are you sure you are using local install of eslint? the cypress plugin should have these globals set